README.md

```diff
@@ -1,53 +1,29 @@
-# The ASC Software Organization Technical Steering Committee
+# The Planetary Software Organization Technical Steering Committee
 
-The ASC Software Organization Technical Steering Committee (TSC) is the technical
-governing body of the ASC Software Organization. It is described in the
+The Planeatry Software Organization Technical Steering Committee (TSC) is the technical
```

Comment:
Planetary

Reply:
Wondering if I could actually be more helpful here and submit a correction myself? Is that easily possible. I guess I would have to check out your patch-3 branch, would I have write rights on that? I guess write rights aren't branch-dependent so this should work... trying it out...

Reply:
That's a great question, that I don't know the answer to.  For these, I've just been using the GitHub web pages for all of the edits I'm doing.  I don't have this repo (or the patches) cloned on my machine.  Maybe I'm doing something wrong?  Totally possible.